app/views/users/backup_code_setup/confirm_delete.html.erb

```diff
@@ -1,6 +1,6 @@
 <% title t('forms.backup_code.confirm_delete') %>
 
-<%= image_tag(asset_url('status/warning.svg'), alt: t('errors.alt.warning'), width: 54, class: 'display-block margin-bottom-4') %>
+<%= render AlertIconComponent.new(icon_name: :warning, class: 'display-block margin-bottom-4 alert-icon') %>
```

Comment:
The `'alert-class'` class would be added by the component itself, so I wouldn't expect to see it here.


```suggestion
<%= render AlertIconComponent.new(icon_name: :warning, class: 'display-block margin-bottom-4') %>
```